lib/elixir/lib/kernel/special_forms.ex

```diff
@@ -1323,6 +1323,19 @@ defmodule Kernel.SpecialForms do
       iex> width
       nil
 
+  An `else` option can be given to handle adjusting what is being returned from
```

Comment:
I think we should make it `## Else clauses` doc section (like in `try` doc), and show what exception will be raised in case of no match.